variable/technology/electricity-grid.yaml

```
@@ -1,25 +1,90 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Cost of Demand-not-served|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
+
+Demand-not-served|Electricity:
+   description: Demand-not-served per region (i.e. aggregate-regions, countries, nuts1, nuts2, nuts3, or nodes).
+   unit: MWh
 
 Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+   description: demand in frequency containment reserve (FCR)
```

Please use a capital letter to start any attributes

```suggestion
   description: Demand in frequency containment reserve (FCR)
```